featurebyte/__init__.py

```diff
@@ -477,6 +479,8 @@ def list_deployments(
     "to_timedelta",
     "Table",
     "TableFeatureJobSetting",
+    "Target",
+    "TargetNamespace",
```

Comment:
Is it intended to expose `TargetNamespace` at top level? `FeatureNamespace` is not listed here.

Reply:
yeap - there's a slight difference there, as users are able to create a target namespace directly.